tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
+envlist =
+   py27,
+   py36,
+   py37
+
+[testenv]
+commands =
+    {envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} bootstrap
+    {envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} install test
+    {envbindir}/test
```

Comment:
(I'm also not too happy about buildout being used inside tox.  Other packages tend to call zope-testrunner directly)